x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/ml_inference_logic.ts

```diff
@@ -254,14 +254,15 @@ export const MLInferenceLogic = kea<
       const {
         addInferencePipelineModal: { configuration, indexName },
         isTextExpansionModelSelected,
-        mlInferencePipeline, // Full pipeline definition
+        mlInferencePipeline, // Full pipeline definition,
```

Comment:
? 🙂

Reply:
heh, I wonder if my IDE did that when I "prettier: fixup this file"

Reply:
```suggestion
        mlInferencePipeline, // Full pipeline definition
```